planetarium/NineChronicles
# Bump `MagicOnion` to `5.0.3`

### Description

1. Bump MagicOnion to 5.0.3
2. Bump NineChronicles.RPC.Shared

### How to test

1. Run local headless with the RPC server feature.
2. Clone NineChronicles repository and checkout.
3. Configure clo.json with your local headless' rpc information.
4. See gRPC communication works.

### Related Links

 - Artifact source: https://github.com/Cysharp/MagicOnion/actions/runs/3947644660 artifact.

### Required Reviewers

### Screenshot
